lib_xud/src/core/XUD_Signalling.xc

```diff
@@ -69,155 +59,46 @@ int XUD_Init()
                 /* Shouldn't expect to get here, but ignore anyway */
                 break;
         }
-
-#else
```

Comment:
I'm a bit surprised that `XUD_Init()` does not contain any XS2-specific code.  Is that intentional or an oversight?

Reply:
HI @mbanth most differences between archs have now been moved to a Hardware Abstraction Layer (see XUD_HAL.xc)

This aids code readability and hopefully simplifies adding new archs in the future.

Reply:
Sounds good, @xross.  Thanks for the explanation.